src/components/home/home.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import './home.css';
```

Comment:
- [ ] Kindly make sure you have the module home.css in your directory to import it. It's actually the reason why your app is crashing
![image](https://user-images.githubusercontent.com/97100091/225730440-11459b4f-1dbc-488c-a407-91ae5b1fe115.png)